devel/mps_load_test.sh

```diff
@@ -208,7 +202,8 @@ cd "$src/devel"
 
 # Build a version of mod_pagespeed with all optimizations enabled, but with
 # a build that includes DCHECKs.
-make -j8 CONF=$compile_mode apache_trace_stress_test_server \
```

Comment:
Hmm, I think there might be inconsistency between apache_debug_install's current and historical meanings that only half-ways works.  Roughly it used to be the apache_debug_ just meant the apache in APACHE_DEBUG_ROOT, with whatever BUILDTYPE= you would pass, and IIRC the apache_release_ targets touched the system apache (probably, since I never used the apache_release_... ones). I am pretty sure the apache_trace_stress_test_server target was written with this "historical" interpretation.

In fact the actual install portion of apache_debug_install works just fine:

apache_debug_install : apache_debug
	install -c $(GIT_BIN)/libmod_pagespeed.so \
	           $(APACHE_DEBUG_MODULES)/mod_pagespeed.so

GIT_BIN = $(GIT_SRC)/out/$(BUILDTYPE)
(Notice also:
 APACHE_DEBUG_MODULES = $(APACHE_DEBUG_ROOT)/modules
 ... which uses the 'old' meaning of debug)

OTOH:
apache_debug : $(HOOKS_STAMP)
	@echo "building Apache module $@ ..."
	cd $(GIT_SRC) && $(MAKE)
	@echo Built mod_pagespeed successfully:
	ls -l $(GIT_DEBUG_BIN)/libmod_pagespeed.so
	@echo To install, type
	@echo "   " cp $(GIT_DEBUG_BIN)/libmod_pagespeed.so \
		$(APACHE_DEBUG_ROOT)/modules/mod_pagespeed.so

... where GIT_DEBUG_BIN is build-type specific:
GIT_DEBUG_BIN = $(GIT_SRC)/out/Debug

One option might be to basically kill every distinction that uses GIT_DEBUG_BIN vs. GIT_RELEASE_BIN, but fixing all the dependencies in scripts/Makefiles/docs might be unreasonably messy.